service/crt/service.go

```diff
@@ -126,10 +120,15 @@ func New(config Config) (*Service, error) {
 // Boot starts the service and implements the watch for the certificate TPR.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		if err := s.tpr.CreateAndWait(); err != nil {
+		err := s.tpr.CreateAndWait()
+		switch {
+		case tpr.IsAlreadyExists(err):
+			s.Config.Logger.Log("info", "certificate third-party resource already exists")
+		case err != nil:
 			panic(fmt.Sprintf("could not create certificate resource: %#v", err))
+		default:
+			s.Config.Logger.Log("info", "successfully created certificate third-party resource")
 		}
```

Comment:
I should say "protects from potential panics", because of passing nil as an argument. If for example errgo.Cause implementation changed to not handle nil or sth. I know it probably won't be the case, but it feels right, yadda yadda yadda :D